rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -102,7 +89,7 @@ def test_ppo_compilation_and_schedule_mixins(self):
                 _enable_learner_api=True,
             )
             .rollouts(
-                num_rollout_workers=1,
+                num_rollout_workers=0,
```

Comment:
Any reason we changed this?

Reply:
This is an artifact of me having an issue with a Ray nightly where I could not use local mode.